Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/model/Patient.java

```diff
@@ -0,0 +1,209 @@
+    private String zip;
+    private String phone;
+    private String dLicense;
+    private String pid;
+    private String patientId;
+    private String domain;
+    private String aaId;
+
+    public List<DocumentMetadataResult> patientDocuments;
+
+    public List<DocumentMetadataResult> getPatientDocuments() {
+        return patientDocuments;
+    }
+
+    public void setPatientDocuments(List<DocumentMetadataResult> patientDocuments) {
+        this.patientDocuments = patientDocuments;
+    }
+
+    public String getFirstName() {
+        return firstName;
+    }
+
+    public void setFirstName(String firstName) {
+        this.firstName = firstName;
+    }
+
+    public String getLastName() {
+        return lastName;
+    }
+
+    public void setLastName(String lastName) {
+        this.lastName = lastName;
+    }
+
+    public String getMiddleName() {
+        return middleName;
+    }
+
+    public void setMiddleName(String middleName) {
+        this.middleName = middleName;
+    }
+
+    public String getBirthDate() {
+        return birthDate;
+    }
+
+    public void setBirthDate(String birthDate) {
+        this.birthDate = birthDate;
+    }
+
+    public String getGender() {
+        return gender;
+    }
+
+    public void setGender(String gender) {
+        this.gender = gender;
+    }
+
+    public String getSsn() {
+        return ssn;
+    }
+
+    public void setSsn(String ssn) {
+        this.ssn = ssn;
+    }
+
+    public String getStreetAddr() {
+        return streetAddr;
+    }
+
+    public void setStreetAddr(String streetAddr) {
+        this.streetAddr = streetAddr;
+    }
+
+    public String getCity() {
+        return city;
+    }
+
+    public void setCity(String city) {
+        this.city = city;
+    }
+
+    public String getState() {
+        return state;
+    }
+
+    public void setState(String state) {
+        this.state = state;
+    }
+
+    public String getZip() {
+        return zip;
+    }
+
+    public void setZip(String zip) {
+        this.zip = zip;
+    }
+
+    public String getPhone() {
+        return phone;
+    }
+
+    public void setPhone(String phone) {
+        this.phone = phone;
+    }
+
+    public String getdLicense() {
+        return dLicense;
+    }
+
+    public void setdLicense(String dLicense) {
+        this.dLicense = dLicense;
+    }
+
+    public String getPid() {
+        return pid;
+    }
+
+    public void setPid(String pid) {
+        this.pid = pid;
+    }
+
+    public String getPatientId() {
+        return patientId;
+    }
+
+    public void setPatientId(String patientId) {
+        this.patientId = patientId;
+    }
+
+    public String getDomain() {
+        return domain;
+    }
+
+    public void setDomain(String domain) {
+        this.domain = domain;
+    }
+
+    public String getAaId() {
+        return aaId;
+    }
+
+    public void setAaId(String aaId) {
+        this.aaId = aaId;
+    }
+
+    public void setEmptyStrings() {
```

Comment:
The only place I see this used is after the creation of a Patient object -- should these statements go in the constructor, or is there a use case where we're want to blank out the Strings but not empty out the Lists?